phpunit/block-supports/typography-test.php

```diff
@@ -679,7 +679,7 @@ public function data_generate_block_supports_font_size_fixtures() {
 			'returns clamp value using custom fluid config' => array(
 				'font_size_value' => '17px',
 				'theme_slug'      => 'block-theme-child-with-fluid-typography-config',
-				'expected_output' => 'font-size:clamp(16px, 1rem + ((1vw - 3.2px) * 0.147), 17px);',
+				'expected_output' => 'font-size:clamp(16px, 1rem + ((1vw - 6.4px) * 0.179), 17px);',
```

Comment:
Is this expected value different now because of the `maxViewportWidth` and `minViewportWidth` values included in the test theme's `theme.json`?

Reply:
Sorry for the late reply. 

Yes that's right.